OpenRA.Mods.Common/UpdateRules/Rules/20210321/UnhardcodeBaseBuilderBotModule.cs

```diff
@@ -11,27 +11,88 @@
 
 using System.Collections.Generic;
 using System.Linq;
-using OpenRA.Mods.Common.Traits;
 
 namespace OpenRA.Mods.Common.UpdateRules.Rules
 {
 	public class UnhardcodeBaseBuilderBotModule : UpdateRule
 	{
+		MiniYamlNode defences;
+
+		// excludes AttackBomber and AttackTDGunboatTurreted
```

Comment:
Comment style - start with a capital letter, end with punctuation.
```suggestion
		// Intentionally excluding AttackBomber and AttackTDGunboatTurreted from the list of traits extending AttackBase because actors with those are not relevant to the changes made here.
```

Reply:
updated